Description: Fix import of mock / mox
 The "from six.moves import mock" fails in Python 3.2, this patch fixes it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-11-12

--- python-oslotest-1.2.0.orig/oslotest/base.py
+++ python-oslotest-1.2.0/oslotest/base.py
@@ -21,7 +21,10 @@ import tempfile
 
 import fixtures
 import six
-from six.moves import mock
+try:
+    from six.moves import mock
+except:
+    import mock
 import testtools
 
 _TRUE_VALUES = ('True', 'true', '1', 'yes')
--- python-oslotest-1.2.0.orig/oslotest/mockpatch.py
+++ python-oslotest-1.2.0/oslotest/mockpatch.py
@@ -16,7 +16,10 @@
 # under the License.
 
 import fixtures
-from six.moves import mock
+try:
+    from six.moves import mock
+except:
+    import mock
 
 
 class PatchObject(fixtures.Fixture):
--- python-oslotest-1.2.0.orig/oslotest/moxstubout.py
+++ python-oslotest-1.2.0/oslotest/moxstubout.py
@@ -16,7 +16,10 @@
 # under the License.
 
 import fixtures
-from six.moves import mox
+try:
+    from six.moves import mox
+except:
+    import mox
 
 
 class MoxStubout(fixtures.Fixture):
--- python-oslotest-1.2.0.orig/tests/unit/test_base.py
+++ python-oslotest-1.2.0/tests/unit/test_base.py
@@ -19,7 +19,10 @@ import os
 import unittest
 
 import six
-from six.moves import mock
+try:
+    from six.moves import mock
+except:
+    import mock
 import testtools
 
 from oslotest import base
--- python-oslotest-1.2.0.orig/tests/unit/test_mockpatch.py
+++ python-oslotest-1.2.0/tests/unit/test_mockpatch.py
@@ -13,7 +13,10 @@
 #    under the License.
 
 
-from six.moves import mock
+try:
+    from six.moves import mock
+except:
+    import mock
 
 from oslotest import base
 from oslotest import mockpatch
